plugins/studymaster-web/Readme.md

```diff
@@ -27,4 +27,16 @@ Example:
 ```VSMMessage#REQUEST#1615553956654#name;surname;weight;gender;graduated#EnterName;EnterSurname;in kg;m,f,d;yes#text;text;text;radio;checkbox```
 
 This example would look like this in the VisualSceneMaker:
-```agent: [REQUEST var='name;surname;weight;gender;graduated' values='EnterName;EnterSurname;in kg;m,f,d;yes' type='text;text;text;radio;checkbox'].```
\ No newline at end of file
+```agent: [REQUEST var='name;surname;weight;gender;graduated' values='EnterName;EnterSurname;in kg;m,f,d;yes' type='text;text;text;radio;checkbox'].```
+
+## How to build react-studymaster in resources
+After building, the folder main\resources\react-studymaster\build\static will be updated.
+
+The JavaScript Library React is being used here, you can find more information here: https://reactjs.org/
+
+To build (use the changes of the folder main\resources\react-studymaster\src) you need to:
+* build a new React App to get the "node_modules" folder by typing into a console: ```npx create-react-app my-app```
+* move the folder "node_modules" inside the folder main/resources/react-studymaster
```

Comment:
You can get the node_modules by running `npm install` in react-studymaster. You don't need to create a new app

Reply:
Indeed. Tested it and updated docs.